modules/core/shared/src/main/scala/scalacache/AbstractCache.scala

```diff
@@ -49,123 +55,92 @@ trait AbstractCache[V] extends Cache[V] with LoggingSupport {
       if (logger.isDebugEnabled) {
         logger.debug(s"Skipping cache PUT because cache writes are disabled. Key: $key")
       }
-      mode.M.pure(())
+      F.pure(())
```

Comment:
`F.unit` ?

Reply:
That'll need `.widen` - F is invariant and we need `F[Any]` unless I change the interface's type. Which I want to do, but maybe as a different change.